.github/workflows/ci.yaml

```diff
@@ -21,7 +21,6 @@ jobs:
         uses: actions/setup-node@master
         with:
           node-version: '${{ steps.console-nvm.outputs.NVMRC }}'
-        working-directory: ./console/wrapper
```

Comment:
I don't know if we want to do this, since each project may have a separate node version it's requesting.

Reply:
GitHub was throwing exceptions saying the `working-directory` options wasn't allowed there. I thought it would still use the correct node version because of `node-version: '${{ steps.console-nvm.outputs.NVMRC }}'`.

Reply:
This has been fixed by separating the CI into jobs for each packages